devito/operator.py

```diff
@@ -82,16 +82,17 @@ def __init__(self, nt, shape, dtype=np.float32, stencils=[],
             dimensions += [i for i in retrieve_dimensions(eqn.rhs) if i not in dimensions]
 
         # Time dimension is fixed for now
-        time_dim = t
+        time_dim = [t, time]
```

Comment:
This is VERY confusing. I have read the entire diff and still don't know which time symbol is which.